### PR TITLE
Cloud November Release Notes

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/CC-3671

Lots of changes this past month so feel free to suggest deletions if there's too much detail.

### CODE REVIEW

**Review by kannanlakshmi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12164)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @gemma-shay, @jramos-crl, and @thtruo)

___
*[releases/cockroachcloud-11082021.md, line 13 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnZAffp8MhghqGFW8hm:-MnZAffq7Nykl4DvtotU:bwr900b) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L13)):*
> ```Markdown
> {% include marketo.html %}
> 
> ### General changes
> ```

Can we add something about Serverless launched besides this one bullet point or do you think this is sufficient? I was thinking we launched CRDB Serverlress, read more here and then keep the remaining sentences you have. 
___
*[releases/cockroachcloud-11082021.md, line 20 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnZAoo052MYDgHiwAg_:-MnZAoo052MYDgHiwAga:bgm5pwl) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L20)):*
> ```Markdown
> - New {{ site.data.products.dedicated }} clusters will now run CockroachDB [v21.1.11](v21.1.11.html).
> 
> ### Console changes
> ```

Did we talk about statements, transactions and sessions pages last month already? I thought that released in Oct?
___
*[releases/cockroachcloud-11082021.md, line 29 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnZAWVl7j08HTQZPRA_:-MnZAWVl7j08HTQZPRAa:bbo7tws) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L29)):*
> ```Markdown
> - The PowerShell command to [download the CockroachDB binary](../cockroachcloud/connect-to-your-cluster.html?filters=windows#step-3-connect-to-your-cluster) is now improved for Windows users.
> - Storage is now show in MiB instead of GiB in the **Storage used** graph on the {{ site.data.products.serverless }} [Cluster Overview](../cockroachcloud/cluster-overview-page.html#storage-used) page.
> - AWS will now be the pre-selected cloud provider on the **Create your cluster** page for most organizations.
> ```

I would remove this. Esp since its "most orgs" its randomized basically with a slight preference for AWS. but adding this raises more questions than answers.

**Comment by kannanlakshmi** [comment]
Also can you add @kevin-v-ngo to the reviewer list? I know its a lot of PMs but for now till we have a better process, let's err on the side of adding more reviewers. 
Kevin - specifically for you, I wasn't sure if we had called our statements, transactions, sessions pages being released already.

**Review by jramos-crl — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12164)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @gemma-shay, @kannanlakshmi, and @thtruo)

___
*[releases/cockroachcloud-11082021.md, line 28 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnaDMykAtw4WAapWBOV:-MnaDMylAqwfP70nc2NU:b4w9ljk) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L28)):*
> ```Markdown
> - Average RU usage is now shown in the **Request Units** chart for the {{ site.data.products.serverless }} [Cluster Overview](../cockroachcloud/cluster-overview-page.html#request-units) page.
> - The PowerShell command to [download the CockroachDB binary](../cockroachcloud/connect-to-your-cluster.html?filters=windows#step-3-connect-to-your-cluster) is now improved for Windows users.
> - Storage is now show in MiB instead of GiB in the **Storage used** graph on the {{ site.data.products.serverless }} [Cluster Overview](../cockroachcloud/cluster-overview-page.html#storage-used) page.
> ```

Two corrections here:
1) Slight type on "Storage is now show in..." should be "shown".
2) It is shown in MiB for the first 1k MiB, after that it turns back to GiB. It goes from 999 MiB / XX GiB to 1 GiB / XX Gib
___
*[releases/cockroachcloud-11082021.md, line 29 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnZAWVl7j08HTQZPRA_:-MnaDfmk7Falt4jQq66e:bv130z5) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L29)):*
<details><summary><i>Previously, kannanlakshmi (Lakshmi Kannan) wrote…</i></summary><blockquote>

I would remove this. Esp since its "most orgs" its randomized basically with a slight preference for AWS. but adding this raises more questions than answers. 
</blockquote></details>

I second this, it is a 75/25 split or so, and mostly for testing purposes. I would remove it too.

**Review by jramos-crl — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12164)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @gemma-shay, @kannanlakshmi, and @thtruo)

___
*[releases/cockroachcloud-11082021.md, line 28 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnaDMykAtw4WAapWBOV:-MnaDtjs7gMlOZ9jq0Oo:bv60y49) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L28)):*
<details><summary><i>Previously, jramos-crl wrote…</i></summary><blockquote>

Two corrections here:
1) Slight type on "Storage is now show in..." should be "shown".
2) It is shown in MiB for the first 1k MiB, after that it turns back to GiB. It goes from 999 MiB / XX GiB to 1 GiB / XX Gib
</blockquote></details>

Ironically I had a typo when describing that there was a typo...I said "type", should be "typo". Oops!

**Review by thtruo — COMMENTED** [review]
> Reviewed 2 of 3 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12164)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @gemma-shay, @kannanlakshmi, and @kevin-v-ngo)

___
*[releases/cockroachcloud-11082021.md, line 33 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnaRZjA8121L93r7qYe:-MnaRZjB1S3lawQBZX9j:b-u9ln2z) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L33)):*
> ```Markdown
> - Previously, clicking **cancel** while editing a cluster would take users back to the **Clusters** page. Now, users are taken back to the cluster's **Overview** page.
> 
> ### Bug fixes
> ```

One suggestion: there was another bug (reported by someone in the community Slack channel) we resolved where RU and SQL Statement graphs stopped updating and polling for new data after a certain point. Could be worth adding a bullet to explicitly call this out. For more details, refer to https://cockroachlabs.atlassian.net/browse/CC-5134 and https://github.com/cockroachlabs/managed-service/pull/6947

**Review by gemma-shay — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12164)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @ianjevans, @jramos-crl, @kannanlakshmi, @kevin-v-ngo, and @thtruo)

___
*[releases/cockroachcloud-11082021.md, line 13 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnZAffp8MhghqGFW8hm:-MnahITBBiSMXjsaSpL5:b-7hux9b) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L13)):*
<details><summary><i>Previously, kannanlakshmi (Lakshmi Kannan) wrote…</i></summary><blockquote>

Can we add something about Serverless launched besides this one bullet point or do you think this is sufficient? I was thinking we launched CRDB Serverlress, read more here and then keep the remaining sentences you have. 
</blockquote></details>

I'm not really sure here, @ianjevans?
___
*[releases/cockroachcloud-11082021.md, line 20 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnZAoo052MYDgHiwAg_:-Mnag-2V0O9Y2nErtmPI:b-trnnpu) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L20)):*
<details><summary><i>Previously, kannanlakshmi (Lakshmi Kannan) wrote…</i></summary><blockquote>

Did we talk about statements, transactions and sessions pages last month already? I thought that released in Oct?
</blockquote></details>

I think those went out in June & July for Dedicated
___
*[releases/cockroachcloud-11082021.md, line 28 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnaDMykAtw4WAapWBOV:-Mnag6Dk3QPxY4GR0XuG:b-896fja) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L28)):*
<details><summary><i>Previously, jramos-crl wrote…</i></summary><blockquote>

Ironically I had a typo when describing that there was a typo...I said "type", should be "typo". Oops! 
</blockquote></details>

Done!
___
*[releases/cockroachcloud-11082021.md, line 29 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnZAWVl7j08HTQZPRA_:-Mnag4LTE0hOyA-ao4_p:b-48vilg) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L29)):*
<details><summary><i>Previously, jramos-crl wrote…</i></summary><blockquote>

I second this, it is a 75/25 split or so, and mostly for testing purposes. I would remove it too.
</blockquote></details>

Good call, done.
___
*[releases/cockroachcloud-11082021.md, line 33 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnaRZjA8121L93r7qYe:-MnahD0g6MrqzcaW4SLQ:b-896fja) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L33)):*
<details><summary><i>Previously, thtruo (Tommy) wrote…</i></summary><blockquote>

One suggestion: there was another bug (reported by someone in the community Slack channel) we resolved where RU and SQL Statement graphs stopped updating and polling for new data after a certain point. Could be worth adding a bullet to explicitly call this out. For more details, refer to https://cockroachlabs.atlassian.net/browse/CC-5134 and https://github.com/cockroachlabs/managed-service/pull/6947
</blockquote></details>

Done!

**Review by thtruo — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12164)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @ianjevans, @jramos-crl, @kannanlakshmi, @kevin-v-ngo, and @thtruo)

___
*[releases/cockroachcloud-11082021.md, line 33 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnaRZjA8121L93r7qYe:-MnbrKeX0b7K3b-wCXjT:bb6ywmx) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L33)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

Done!
</blockquote></details>

LGTM

**Comment by kevin-v-ngo** [comment]
Hi @stbof and @ianjevans, how are we handling release notes and documentation between the cc console and db console? CRDB-2505

**Review by gemma-shay — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12164)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @ianjevans, @jramos-crl, @kannanlakshmi, @kevin-v-ngo, and @thtruo)

___
*[releases/cockroachcloud-11082021.md, line 20 at r2](https://reviewable.io/reviews/cockroachdb/docs/12164#-MnZAoo052MYDgHiwAg_:-MnfoO6kDPdYSkFiOZP4:b-19szil) ([raw file](https://github.com/cockroachdb/docs/blob/e34473d33cbce22419c792139a6baba57b3087d5/releases/cockroachcloud-11082021.md#L20)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

I think those went out in June & July for Dedicated
</blockquote></details>

Ah you meant Serverless, duh. Yes, will add.

File: releases/cockroachcloud-11082021.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{% include marketo.html %}
+
+### General changes
+
+- {{ site.data.products.serverless }}, a fully-managed, auto-scaling deployment of CockroachDB, is now available. To get started with {{ site.data.products.serverless }} for free, see the [Quickstart](../cockroachcloud/quickstart.html).
+- CockroachCloud Free (beta) and CockroachCloud are now {{ site.data.products.serverless }} and {{ site.data.products.dedicated }}, respectively. Your ability to use your cluster will not be affected.
+- {{ site.data.products.serverless }} clusters will now run CockroachDB [v21.2.0-beta.4](v21.2.0-beta.4.html).
+- New {{ site.data.products.dedicated }} clusters will now run CockroachDB [v21.1.11](v21.1.11.html).
+
+### Console changes
```
Comment (kevin-v-ngo) [inline releases/cockroachcloud-11082021.md:20]: Hi @gemma-shay,

I think we need something along the lines of (feel free to update): 

CockroachDB Serverless now supports monitoring workload activity by introducing Sessions, Transactions, and Statements pages under the SQL Activity navigation menu. Statements and Transaction statistics are also persisted on disc allowing longer retention for monitoring activity and performance trends over time.

I'm following up with the team why this was not captured via the release notes generator script.

File: releases/cockroachcloud-11082021.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: What's New in CockroachDB Cloud
+toc: true
+summary: Additions and changes in CockroachDB Cloud since October 3, 2021 .
```
Comment (stbof) [inline releases/cockroachcloud-11082021.md:4]:
```suggestion
summary: Additions and changes in CockroachDB Cloud since October 3, 2021.
```

**Review by ianjevans — APPROVED** [review]

**Review by kevin-v-ngo — APPROVED** [review]

**Pull request merged**